build.gradle.kts

```diff
@@ -140,3 +140,55 @@ val disableSigning by tasks.registering {
 }
 
 tasks.getByName("publishToMavenLocal").dependsOn(disableSigning)
+
+/**
+ * Downloads the BCP 47 Language Tag Registry as defined by iana,
+ * @see <a href="https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry">https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry</a>,
+ * is JSON format.
+ * The types which are required for the validation of the Locale Constraint, are extracted and written to a javascript file.
+ */
+val downloadBCP47LanguageTagRegistry = tasks.register( "downloadBCP47LanguageTagRegistry" ) {
+    val languageTagRegistryScriptFile = file( "src/main/resources/bamm/scripts/languageRegistry.js" )
+    val languageTagRegistryUrl = uri( "https://raw.githubusercontent.com/mattcg/language-subtag-registry/master/data/json/registry.json" ).toURL()
```

Comment:
Should we rely on this "arbitrary" third party repository here? Do we know how up-to-date/well-maintained it is? The alternatives would be:
1. this
2. Download the [primary IANA source](http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry) instead, with the additional parsing overhead
3. Check in either one of the sources (IANA or JSON version) here

Reply:
Looking at the commit history, the repository is updated once a year. I tried to find out how often the IANA source changes, but unfortunately I have not been able to find any concrete information.

How about we keep using this and setup a script which which checks regularly whether there are any changes. In case we find missing changes, we can create a PR and contribute the missing changes?

Reply:
Alright, we can do that if it becomes necessary. I think for now the current implementation suffices.